packages/hmr/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import { container, Piece, Store } from '@sapphire/framework';
+import { watch } from 'chokidar';
+import { relative } from 'path';
+
+/**
+ * Starts HMR for all registered {@link Store Stores} in {@link container.stores the main container}.
+ */
+export function start() {
+	container.logger.info('HMR is enabled!');
```

Comment:
Honestly, for logs from the plugin use a header like `HMR-Plugin: Enabled HMR.` or w/e